contextproject-ms3/src/main/java/contextproject/App.java

```diff
@@ -1,5 +1,8 @@
 package contextproject;
 
+
+import contextproject.controllers.CLIController;
+
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
```

Comment:
Checkstyle complains about this (wrong lexicographical order), could you fix this?